external/fv3fit/tests/training/test_train.py

```diff
@@ -123,6 +123,10 @@ def get_dataset(model_type, sample_func):
         )
     else:
         output_values = input_values
+        # add an input with feature dim size 1
+        input_variables.append("var_in_2d")
```

Comment:
nit: instead of modifying input_variables here, can you update the hard-code above to be `input_variables=["var_in", "var_in_2d"]`? Then you can clip input_values[-1] below. Alternatively, add a comment above that says `# "var_in_2d" is appended below`